models/intel/person-detection-0100/description/person-detection-0100.md

```diff
@@ -0,0 +1,55 @@
+# person-detection-0100
+
+## Use Case and High-Level Description
+
+This is a pedestrian detector for the Retail scenario. It is based on MobileNetV2
+backbone with two SSD heads from 1/16 and 1/8 scale feature maps and clustered
+prior boxes for 256x256 resolution.
+
+## Example
+
+![](./person-detection-0100.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP                              | 82.52%                                    |
+| Pose coverage                   | Standing upright, parallel to image plane |
+| Support of occluded pedestrians | YES                                       |
+| Occlusion coverage              | <50%                                      |
+| Min pedestrian height           | 100 pixels (on 1080p)                     |
+| GFlops                          | 0.786                                     |
+| MParams                         | 1.817                                     |
+| Source framework                | Caffe*                                    |
```

Comment:
```suggestion
| Source framework                | Pytorch*                                    |
```

Reply:
I see in internal repo that this framework was used, if not please discard my suggestion